config/crd/kustomization.yaml

```diff
@@ -8,6 +8,11 @@ resources:
   - bases/tinkerbell.org_workflowdata.yaml
 #+kubebuilder:scaffold:crdkustomizeresource
 
+# Labels to add to all resources and selectors.
+commonLabels:
```

Comment:
Given recent discussions do we still want this? Is it going to apply it to all CRDs? I suspect we don't want to move Workflow's given they're ephemeral.